examples/systemcalls/systemcalls.c

```diff
@@ -84,14 +151,61 @@ bool do_exec_redirect(const char *outputfile, int count, ...)
     // and may be removed
     command[count] = command[count];
 
+    /*
+     * TODO
+     *   Call execv, but first using https://stackoverflow.com/a/13784315/1446624 as a refernce,
+     *   redirect standard out to a file specified by outputfile.
+     *   The rest of the behaviour is same as do_exec()
+     *
+     */
+    pid = fork();
```

Comment:
@Induja21 think about how you coudl avoid duplicating logic with the function above.